tools/src/main/java/com/nedap/archie/adl14/ADL14Converter.java

```diff
@@ -20,23 +23,44 @@
 
     private final MetaModels metaModels;
     private final ADL14ConversionConfiguration conversionConfiguration;
+    InMemoryFullArchetypeRepository existingRepository;
 
     public ADL14Converter(MetaModels metaModels, ADL14ConversionConfiguration conversionConfiguration) {
         this.metaModels = metaModels;
         this.conversionConfiguration = conversionConfiguration;
     }
 
 
+    public void setExistingRepository(InMemoryFullArchetypeRepository existingRepository) {
+        this.existingRepository = existingRepository;
+    }
```

Comment:
This repository could be modified by the `convert` method. Is that expected?

Reply:
I'll add comment